R/get_sard.R

```diff
@@ -0,0 +1,57 @@
+#' @rdname db_getter_backend
+#' @export
+get_sard <- function(db_url = get_db_url("sard")) {
+
+  # read data
+  sard <- db_url %>%
+    data.table::fread(
+      drop = c(),
+      colClasses = c(
+        "Lab ID" = "character",
+        "Date" = "character",
+        "Uncertainty" = "character",
+        "delta 13" = "character",
+        "Species" = "character",
+        "Material dated" = "character",
+        "Dating technique" = "character",
+        "Site" = "character",
+        "Country" = "character",
+        "Stratigraphic context" = "character",
+        "Archaeological Period" = "character",
+        "Archaeological Sub-chronology" = "character",
+        "DecdegS" = "character",
+        "DecdegE" = "character",
+        "Notes" = "character",
+        "refcode" = "character"
+      ),
```

Comment:
`colClasses = "character"` is sufficient, as I recently learned.

Reply:
This makes it way easier indeed 🥳